versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.15"
+      version = ">= 3.42"
```

Comment:
looks like this is failing the checks

Reply:
Interesting. I can see that it's failing, but not why 🤔

Reply:
I updated it to 3.42, because it included a bug fix for the networkfirewall_rule_group, which allowed updating the rule group.

Reply:
i see, if its not blocking maybe don't change this first? think there could be something that is not compatible, but not related to our changes

Reply:
OK, I'm rolling this back to 3.15 for now.

Reply:
Ah, I think I figured it out. The default version of Terraform that the actions are using is `0.12.26`. This warning is key
> Warning: registry.terraform.io: This version of Terraform has an outdated GPG key and is unable to verify new provider releases. Please upgrade Terraform to at least 0.12.31 to receive new provider updates. For details see: https://discuss.hashicorp.com/t/hcsec-2021-12-codecov-security-event-and-hashicorp-gpg-key-exposure/23512

Reply:
I have a PR up to fix the underlying cause of the test failures in the upstream repo: https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/678